src/libs/actions/PersonalDetails.js

```diff
@@ -420,6 +421,44 @@ function clearAvatarErrors() {
     });
 }
 
+/**
+ *
+ * @param {Object} detail
+ * @returns {Object}
+ */
+function buildNewPersonalDetail(detail) {
```

Comment:
Can we rename this to avoid confusion between the singular and plural method names?